app/src/main/java/com/calvinlsliang/simpletodo/EditItemActivity.java

```diff
@@ -0,0 +1,63 @@
+package com.calvinlsliang.simpletodo;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.EditText;
+
+public class EditItemActivity extends AppCompatActivity {
+
+    private final int RESULT_OK = 40;
+    int pos;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_edit_item);
+
+        String item = getIntent().getStringExtra("item");
```

Comment:
Make these keys public static constants so that both activities can guarantee theyre using the same keys